tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "include": ["./*", "src", "testUtils", "types", "deployment"],
+  "include": ["./*", "src", "testUtils", "types", "deployment", "remix.init"],
```

Comment:
Can't have this here unless you remove it with the initscript. The final repo after the create-remix run will not have the remix.init folder